proto/initia/reward/v1/query.proto

```
@@ -1,6 +1,8 @@
 syntax = "proto3";
 package initia.reward.v1;
 
+import "amino/amino.proto";
```

Buf (1.47.2)

import "amino/amino.proto": file does not exist

(COMPILE)